airbyte-integrations/bases/source-acceptance-tests/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM openjdk:14.0.2-slim
+
+# Install Docker to launch worker images. Eventually should be replaced with Docker-java.
+# See https://gitter.im/docker-java/docker-java?at=5f3eb87ba8c1780176603f4e for more information on why we are not currently using Docker-java
+RUN apt-get update && apt-get install -y \
+                          apt-transport-https \
+                          ca-certificates \
+                          curl \
+                          gnupg-agent \
+                          software-properties-common
+RUN curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add -
+RUN add-apt-repository \
+       "deb [arch=amd64] https://download.docker.com/linux/debian \
+       $(lsb_release -cs) \
+       stable"
+RUN apt-get update && apt-get install -y docker-ce-cli jq
```

Comment:
for my own understanding, all the apt-gets before this comment are for installing docker?

Reply:
gotcha. don't need to spend time if all of this is going to go away. I was personally curious